test/command_callback/test_slimlint_command_callback.vader

```diff
@@ -0,0 +1,18 @@
+Before:
+  runtime ale_linters/slim/slimlint.vim
+
+  call ale#test#SetDirectory('/testplugin/test/command_callback')
+
+After:
+  Restore
+  call ale#linter#Reset()
+  call ale#test#RestoreDirectory()
+
+Execute(The command should be correct without a .rubocop.yml):
+  AssertEqual 'slim-lint %t', ale_linters#slim#slimlint#GetCommand(bufnr(''))
+
+Execute(The .rubocop.yml should be detected correctly):
+  call ale#test#SetFilename('../slimlint-test-files/subdir/file.slim')
```

Comment:
@w0rp can you help me here? I'm really struggling to get this test to work, even after looking at other tests. How can I set the directory/file for `bufnr`?

Reply:
Hmm wait, it is failing on CI like expected, but here locally it does not fail ... weird. Looks like this is the right way then.